src/gluonts/nursery/autogluon_tabular.py

```diff
@@ -103,21 +103,34 @@ def to_forecast(
 
 
 class TabularEstimator(Estimator):
-    def __init__(self, freq: str, prediction_length: int) -> None:
+    def __init__(self, freq: str, prediction_length: int, **kwargs) -> None:
         super().__init__()
         self.task = task
         self.freq = freq
         self.prediction_length = prediction_length
+        default_kwargs = {
+            "excluded_model_types": ["KNN", "XT", "RF"],
+            "hyperparameters": "light",
+            "num_bagging_sets": 1,
```

Comment:
I would run a small set of benchmarks to see whether this default_kwarg is better (in terms of forecast-accuracy, disk space used) than an alternative version of default_kwarg where you instead delete: `hyperparameters`, `num_bagging_sets`, 
and replace 
`presets: [ "high_quality_fast_inference_only_refit", "optimize_for_deployment",]`

Reply:
Working on it. Should `eval_metric="mean_absolute_error"` also be the hard-set argument instead of being in the kwargs?

Reply:
No that should be left as default kwarg, as some users may want to change its value